src/AppBundle/Entity/Client.php

```diff
@@ -288,13 +303,13 @@ public function getFullname()
 
     public function setCourtDateWithoutTime($courtDate = null)
     {
-        $this->courtDate = ($courtDate instanceof \DateTime) ?
-                new \DateTime($courtDate->format('Y-m-d')) : null;
+        $this->courtDate = ($courtDate instanceof DateTime) ?
+                new DateTime($courtDate->format('Y-m-d')) : null;
     }
 
     public function isValidCourtDate(ExecutionContextInterface $context)
     {
-        $today = new \DateTime();
+        $today = new DateTime();
```

Comment:
@slizzio @iterative-it  I think we need to agree on what to do with fully qualified namespaces vs imported ones for PHP native classes. Most of the code is  using "\DateTime" without importing the namespace in other entities ATM